openid-4-verifiable-presentations-1_0.md

```diff
@@ -1704,6 +1704,17 @@ Note: If the Verifier's Response URI did not return a `redirect_uri` in step (6)
 
 (10) The Verifier checks whether the `nonce` received in the Credential(s) in the VP Token in step (9) corresponds to the `nonce` value from the session. The Verifier then consumes the VP Token and invalidates the `transaction-id`, `request-id` and `nonce` in the session.
 
+## Pre-Final Specifications
+
+Implementers should be aware that this specification uses several specifications that are not yet final specifications. Those specifications are:
+
+- OpenID Federation 1.0 draft -42 [@!OpenID.Federation]
+- SIOPv2 draft -13 [@!SIOPv2]
+- SD-JWT-based Verifiable Credentials (SD-JWT VC) draft -08 [I-D.ietf-oauth-sd-jwt-vc]
```

Comment:
Should a specific version of SD-JWT be listed too? I don't think SD-JWT VC refers to a specific draft.

Reply:
So we're saying that for SD-JWT we *do* want people to use the latest rather than a particular numbered draft? That seems strange to me. I think we should explicitly state a version for SD-JWT too.

Reply:
I wondered about that as well, but openid connect core spec worked the same way: The reference was to latest (stable url to latest), but the note then specified specific draft versions with this note.

--> This is why i only adjusted the text here and left the references to latest.

Reply:
we do not reference sd-jwt spec from openid4vp, only sd-jwt vc.
plus, this is what i meant - no need for a draft version for sd-jwt vc because ietf tooling will give the latest one:
```suggestion
- SD-JWT-based Verifiable Credentials (SD-JWT VC) draft [@!I-D.ietf-oauth-sd-jwt-vc]
```

Reply:
I would still provide that draft version in the section explicitly if you don't mind - seems cleaner to me to have that in text for all non-final specs

Reply:
I thought the point was to refer to explicit versions, as Christian had done, so that if breaking changes are made it's clear which one should be followed to be conformant.

If that is the case I think we need to specify explicit versions of both SD-JWT VC and SD-JWT to achieve the stated aim.

(Although equally it is going to be confusing if we're saying "use draft 8" alongside a link to (say) draft 9. I guess that was Kristina's point...)